README.md

```diff
@@ -25,6 +21,34 @@ This guide will show you how to:
 
 [Try out the example.](https://alphagov.github.io/govuk-country-and-territory-autocomplete/examples/)
 
+---
+
+## Support
+
+The GOV.UK Design System team maintains the country and territory autocomplete. However, we’re only able to put in minimal work to support it.
+
+[Read about our plans to maintain this component.](LINK-TO-PINNED-RESEARCH-ISSUE)
```

Comment:
I think the style guide says we should put full stops outside of links?

```suggestion
[Read about our plans to maintain this component](LINK-TO-PINNED-RESEARCH-ISSUE).
```

Reply:
Weirdly it's in the [Design System style guide](https://design-system.service.gov.uk/styles/typography/#links), but not the [GOV.UK style guide](https://www.gov.uk/guidance/content-design/links) 🤔 Is there a disagreement here, or just one of those weird overlaps?

Reply:
I think where the link is the entire sentence, you can link it - especially if separate from other paragraphs.

Reply:
To stay consistent with what we've done elsewhere in Design System content, might make sense to put full stops outside the links. For example, the way we've displayed the link text in the What's New content on our [homepage](https://design-system.service.gov.uk/).

Worth looping @calvin-lau-sig7 into this convo...

Reply:
The guidance on the [Typography](https://design-system.service.gov.uk/styles/typography/#links) page? I'd say it covers 'services' but not 'flat content'... I know the boundary can be fuzzy sometimes!

Reply:
@calvin-lau-sig7 Yeah, exactly. Was thinking the README falls under 'flat content'.

Reply:
Yeah, readme as 'flat content' sounds right, up to you of course!

I think a full sentence is fine to include the full stop — and that's how I'd interpret the guidance in Typography. As in, 'at the end' means the same as 'the last part.'

> If your link is at the end of a sentence or paragraph, make sure that the linked text does not include the full stop.

Not sure there's a screen reader consideration or something I don't know about? Otherwise I might go and tweak the guidance in 'Typography.'

Reply:
@calvin-lau-sig7 I've been looking at our READMEs and release notes for entire sentences that are links. Since this content is also on GitHub, it seems like a fair point of comparison. Looks like our custom is, just like Design System guidance and Frontend docs, to leave the full stop unlinked. So maybe we can stay consistent with that and go with @lfdebrux's suggested changes.